psi4/src/psi4/dfocc/dfocc.cc

```diff
@@ -126,7 +126,7 @@ void DFOCC::common_init() {
             }
             tol_grad = pow(10.0, -temp);
             // tol_grad = 100.0*tol_Eod;
-            outfile->Printf("\tRMS orbital gradient is changed to : %12.2e\n", tol_grad);
+            outfile->Printf("\tFor this energy convergence, default RMS orbital gradient is: %12.2e\n", mograd_max);
```

Comment:
isn't the var `tol_grad` still?

Reply:
Clearly, my attempts to be careful when copy-pasting are not careful enough... Fixing.